cfgov/jinja2/v1/landing-page/index.html

```diff
@@ -136,4 +136,13 @@ <h2 class="o-half-width-link-blob-group__header">
             {{ render_stream_child(block) }}
         {% endif %}
     {% endfor %}
+
+    {% if page.email_signup or prototype %}
+        {% if prototype %}
```

Comment:
No just here. It looks like this doesn't have the updated landing-page/index.html, which should have an example of this.